docs/groups.md

```diff
@@ -24,7 +24,7 @@ We can create groups by creating a new file in the `groups` directory:
 description: My exclusion group
 
 slots:
-  - feature: firstFeature  # referring features/firstFeature.yml
```

Comment:
the extra space was there to keep same vertical alignment for all comments inside this code block.

but I understand it can be very difficult to maintain for automated formatting. do you have any opinions about it regarding keeping formatting optional for some cases?

Reply:
Considering all issues with Markdown formatting, I think it's easier to simply ignore *.md files from Prettier.

Reply:
yeah, we can take it in steps. 

initially we can get the Prettier/ESLint config in targeting only one package first, and then once we are happy with the overall config setup and how it's applied to one package, we can merge the PR.

and then we can send individual PRs targeting individual packages, with docs as the last thing we might want to tackle.